app_config.tf

```diff
@@ -0,0 +1,15 @@
+module app_config {
+  source   = "./modules/databases/app_config"
+  for_each = local.database.app_config
+  name     = each.value.name
+
+  location            = lookup(each.value, "region", null) == null ? module.resource_groups[each.value.resource_group_key].location : local.global_settings.regions[each.value.region]
+  resource_group_name = module.resource_groups[each.value.resource_group_key].name
+  global_settings     = local.global_settings
+  settings            = each.value
+}
+
+output app_config {
+  value     = module.app_config
+  sensitive = true
```

Comment:
@erikschlegel given the CAF team's guidance on this flag do we still want to move forward with it?  They have taken it out of the core modules but use it in the landing zone itself

Reply:
Makes me really nervous getting rid of the sensitive flag given the cosmos connection string would be exposed in clear text within the state file. Removing this flag seemed like a band aid fix due to upgrading to the latest version of Terraform.

Reply:
agreed, just wanted to verify